bin/submarine-daemon.sh

```diff
@@ -36,9 +36,9 @@ cd ${BIN}/>/dev/null
 SUBMARINE_SERVER_NAME="Submarine Server"
 SUBMARINE_SERVER_LOGFILE="${SUBMARINE_LOG_DIR}/submarine.log"
 SUBMARINE_SERVER_MAIN=org.apache.submarine.server.SubmarineServer
-JAVA_OPTS+=" -Dsubmarine.log.file=${SUBMARINE_SERVER_LOGFILE}"
+JAVA_OPTS+="${SUBMARINE_SERVER_JAVA_OPTS} ${SUBMARINE_SERVER_MEM} -Dsubmarine.log.file=${SUBMARINE_SERVER_LOGFILE}"
 
-add_jar_in_dir "${BIN}/../lib"
+add_each_jar_in_dir_recursive "${BIN}/../lib"
```

Comment:
maybe create a variable SUBMARINE_HOME=${BIN}/.. and then add_each_jar_in_dir_recursive "${SUBMARINE_HOME}/lib"  is better?

Reply:
`SUBMARINE_HOME` already exist in `conf/submarine-env.sh`

Reply:
For now, let's just use ${BIN}/../lib, as I'm not sure about the conflicts with SUBMARINE_HOME in submarine-env.sh
@hhhizzz, If you have the bandwidth, help us to optimize it~